Sources/Purchasing/Purchases/PurchasesType.swift

```diff
@@ -26,6 +26,14 @@ public protocol PurchasesType: AnyObject {
      */
     var appUserID: String { get }
 
+    /**
+     * The storefront country code currently used by the Apple account. This is always provided by StoreKit 1.
+     * #### Related articles
+     * - ``Purchases/getStorefront(completion:)``
+     * - ``Purchases/getStorefront()``
+     */
+    var storeFrontCountryCode: String? { get }
```

Comment:
The documentation of this in `Purchases` is 
```
    /// The three-letter code representing the country or region
    /// associated with the App Store storefront.
    /// - Note: This property uses the ISO 3166-1 Alpha-3 country code representation.
```
I don't know if we should unify them both as customers will only see one of the two.